### PR TITLE
React build

### PR DESCRIPTION
**Pull request by dflynn15**
- Adds ESLint for JS Lint checking from SB's standard package
- Adds dependencies required for the React script
- Adds Yarn documentation 💻 🐈 💨 ✨ 
- Moves `.babelrc` to root directory for other JS files to utilize in other development

@bryanbraun 🙈

### CODE REVIEW

**Comment by dflynn15** [comment]
Friendly ☎️  to see if this is something we want to pursue.

**Comment by bryanbraun** [comment]
@dflynn15, ah yes... thanks for the reminder. Yes, these are features I'm interested in. I'm hoping to get the time to look closely at them tomorrow. 👍

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "sparkbox"
```
Comment (bryanbraun) [inline .eslintrc]: Whoops. Looks like I need to pull in the NPM dependencies for this, per [the README](https://github.com/sparkbox/eslint-config-sparkbox#eslint-config-sparkbox).

Simple enough. I'll just tack that onto a commit after merging this.

File: package.json [hunk 2]
```diff
@@ -26,12 +27,17 @@
   },
   "homepage": "https://github.com/sparkbox/bouncy-ball#readme",
   "dependencies": {
+    "eslint-config-sparkbox": "^0.1.0",
```
Comment (bryanbraun) [inline package.json]: I'll move this down to devDependencies, when I make the other dependency changes. 👍

**Pull request merged**